frontend/src/modules/scribble/components/RoundNumber.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+    <div class="round-number">
+        <b-container fluid>
+            <b-row align-v="center" class="justify-content-md-center">
+                <b-col md="auto">Round: </b-col>
+                <b-col md="auto"
+                    ><input
+                        class="round-number__input"
+                        type="text"
+                        v-model="roundNum"
+                        readonly
+                /></b-col>
+            </b-row>
+        </b-container>
+    </div>
+</template>
+
+<script>
+export default {
+    name: "RoundNumber",
+
+    props: {
+        roundNumber: Number,
+    },
+
+    data: function () {
+        return {
+            roundNum: this.roundNumber + 1,
```

Comment:
nit: I would add a comment or use a variable for the 1- the +1 is not obvious to why from the code.

Reply:
As a note, programmers have a term for this, magic numbers! https://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad